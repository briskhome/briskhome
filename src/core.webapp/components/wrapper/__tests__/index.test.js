import React from 'react';
import { shallow } from 'enzyme';

import Wrapper from '../';

describe('core.webapp -> components -> wrapper', () => {
  it('renders', () => {
    expect(shallow(<Wrapper>Lorem ispum</Wrapper>)).toMatchSnapshot();
  });
});
