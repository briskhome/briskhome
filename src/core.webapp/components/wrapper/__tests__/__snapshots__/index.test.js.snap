// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core.webapp -> components -> wrapper renders 1`] = `
<React.Fragment>
  <Connect(Navigation) />
  <section>
    <Switch>
      Lorem ispum
    </Switch>
  </section>
</React.Fragment>
`;
